terraform/modules/hub/kms.tf

```diff
@@ -85,6 +85,7 @@ data "aws_iam_policy_document" "policy" {
       identifiers = [
         "arn:aws:iam::${data.aws_caller_identity.account.account_id}:root",
         "arn:aws:iam::${data.aws_caller_identity.account.account_id}:role/${var.deployment}-policy-execution",
+        "arn:aws:iam::${data.aws_caller_identity.account.account_id}:role/${module.policy_fargate.execution_role_name}",
```

Comment:
Can we add depends_on `module.policy_fargate.execution_role_name` to test?

Reply:
I really don't think it will do anything, the dependency is almost certainly already there.  But I'll add the line anyway just to see.